src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -207,4 +292,40 @@ private String getRulesText() {
   private void setRules(String rules) {
     circulationRulesFixture.updateCirculationRules(rules);
   }
+
+  private Set<UUID> getSetOfPolicyIds(int numberOfPolicies) {
+    Set<UUID> ofpIds = new HashSet<>();
```

Comment:
Can you replace `ofpIds` with a better name? For example `ids`?

Reply:
renamed